src/EventListener/BreadcrumbListener.php

```diff
@@ -48,7 +48,7 @@ public function __construct(Reader $reader, Trail $breadcrumbTrail)
     public function onKernelController(KernelEvent $event)
     {
         if (!is_array($controller = $event->getController())) {
-            return;
+            $controller = [$controller, '__invoke'];
```

Comment:
What if controller isn't an array, but is something different as a callable?

Would an early return in case it isn't callable make sense?

```
if (!is_callable($controller)) {
    return;
}
```

Reply:
Both constructor and setter for both events have `callable` type-hint for `$controller`, so I guess there isn't any point in such a check

Reply:
The check wouldn't be needed for `$controller`, but on the combination of `[$controller, '__invoke']` as not every controller does implement that.

Reply:
Added more callable type checks for what can not be handled with Doctrine annotation reader along with this check